### PR TITLE
fix(perception): adopt awsim (tlr) camera topic

### PR DESCRIPTION
**Pull request by MasatoSaeki**
## Description
This PR enable to adopt awsim camera topic.
link
- https://github.com/autowarefoundation/autoware.universe/pull/8676
- https://github.com/autowarefoundation/autoware_launch/pull/1144

## Tests performed
I have checked on AWSIM

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: autoware_launch/launch/e2e_simulator.launch.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
   <arg name="lanelet2_map_file" default="lanelet2_map.osm" description="lanelet2 map file name"/>
   <arg name="pointcloud_map_file" default="pointcloud_map.pcd" description="pointcloud map file name"/>
   <!-- Perception -->
-  <arg name="traffic_light_namespace" default="traffic_light" description="traffic light recognition namespace1"/>
+  <arg name="traffic_light_namespace" default="[traffic_light]" description="traffic light recognition namespace"/>
```
Comment (YoshiRi) [inline autoware_launch/launch/e2e_simulator.launch.xml:28]: Does this line means we can set namespace array for each cameras?

Comment (MasatoSaeki) [reply autoware_launch/launch/e2e_simulator.launch.xml:28]: Yes. We can set any (and multi) camera namespace for tlr.

**Comment by xmfcx** [comment]
I've rebased this to the latest main. Now testing with [AWSIM Labs](https://autowarefoundation.github.io/AWSIM-Labs/main/). https://github.com/autowarefoundation/AWSIM-Labs/releases

**Comment by xmfcx** [comment]
Works with the latest AWSIM Labs :+1:

**Review by xmfcx — APPROVED** [review]

**Comment by xmfcx** [comment]
@MasatoSaeki -san please next time test with AWSIM Labs.

AWSIM Labs is much more advanced, faster, lighter and with more features than the original AWSIM.

And it works with the main branch of the Autoware.

- https://github.com/autowarefoundation/AWSIM-Labs/releases
- https://autowarefoundation.github.io/AWSIM-Labs/main/
- https://autowarefoundation.github.io/AWSIM-Labs/main/GettingStarted/QuickStartDemo/

**Pull request merged**